lib/modules/datasource/conan/common.ts

```diff
@@ -1,15 +1,9 @@
-import { regEx } from '../../../util/regex';
 import type { ConanPackage } from './types';
 
 export const defaultRegistryUrl = 'https://center.conan.io/';
 
 export const datasource = 'conan';
 
-export const conanDatasourceRegex = regEx(
```

Comment:
i don't understand the issue here, so we need that to be inline?

Reply:
Yes, every global regex should start from the clean state. I'm thinking of our parsing library for easy production of project-specific `eslint` rules like that, by the way.